🦉 **Athena triage note — rebase artifact detected**

All 6 CI failures on this PR stem from a single root cause:

```
src/__tests__/Layout.mobile-responsive.test.tsx(11,26): error TS6133: 'act' is declared but its value is never read.
```

**Fix needed (2 steps):**
1. Remove unused `act` import on line 11 of `Layout.mobile-responsive.test.tsx`
2. Add `approved-minor-bump` label (required by feat: PR gate)

After fix: dashboard-test ✅, dashboard-e2e ✅, lint ✅ already pass. This should go fully green.

cc OneStepAt4time

--- a/dashboard/src/__tests__/Layout.mobile-responsive.test.tsx
+++ b/dashboard/src/__tests__/Layout.mobile-responsive.test.tsx
@@ -8,7 +8,7 @@
  */
 
 import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
-import { render, screen, act } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import { MemoryRouter, Route, Routes } from 'react-router-dom';
 
 const mockSubscribeGlobalSSE = vi.fn();

